www/html/ci/application/controllers/Landing.php

```diff
@@ -34,10 +35,50 @@ public function __construct() {
 		$this->loader = new Twig_Loader_Filesystem('ci/application/views');
 		$this->twig = new Twig_Environment($this->loader);
 	}
+	
+	public function trackingnavigation(){
+		
+       
+		
+		$pagename =  $this->input->post('page');
+		$time =  $this->input->post('time');
+		$city =  $this->input->post('city');
+		$country =  $this->input->post('country');
+		
+		$data = array(
+	        'page_name' => $pagename,
+	        'recorded_at' => $time,
+	        'city' => $city,
+	        'country' => $country
+        );
+
+        $this->db->insert('user_tracking_timing', $data);
+	}
+	public function trackinglanding(){
+		
+	
+		
+	
+	
+		$time =  $this->input->post('time');
+		$city =  $this->input->post('city');
+		$country =  $this->input->post('country');
+		
+		$data = array(
+	   
+	        'recorded_at' => $time,
+	        'city' => $city,
+	        'country' => $country
+        );
+
+        $this->db->insert('user_tracking_entry', $data);
```

Comment:
By MVC structure, any direct data related operations must be done in the Model. 
This includes this line where you call $this->db->insert...
In general, db should not be seen in the controller.

Create a model which has a function to update the database. This seems kind of useless since the controller is basically calling a function to do what it could do by itself, but MVC does not allow this. Controllers are what connects the user experience (views) to the data (models)